extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -21,10 +21,11 @@
           "title": "Interaction Type",
           "type": "string",
           "description": "Specify action made by user on a message.",
-          "enum": ["click", "open", "action"],
+          "enum": ["click", "open", "action", "unsubscribe"],
```

Comment:
We can remove "action" from the enum list as is not getting used.

Reply:
@sawhneyh I have removed "action" now